Library/Formula/tesseract.rb

```diff
@@ -24,10 +25,21 @@ class Tesseract < Formula
   end
 
   option "all-languages", "Install recognition data for all languages"
+  option "with-training-tools", "Install OCR training tools"
+  option "with-opencl", "Enable OpenCL support"
 
   depends_on "libtiff" => :recommended
   depends_on "leptonica"
 
+  if build.with? "training-tools"
```

Comment:
How widely used is this likely to be? I'm a little bleh about introducing so many optional dependencies here unless it's going to prove popular/widely useful/etc.